### PR TITLE
Allow translating stories with lokalise

### PR DESCRIPTION
**Pull request by palmenhq**
need https://github.com/HedvigInsurance/textkeyfy/pull/17 before we can merge this one

### CODE REVIEW

File: src/Components/Common.tsx [hunk 1]
```diff
@@ -15,24 +19,21 @@ export interface Replacements {
 
 export const TranslationNode: React.SFC = ({ children }) => <>{children}</>
```
Comment (styrelius) [inline src/Components/Common.tsx:20]: I think `SFC` is deprecated and that they want you to use `React.FC` or `React.FunctionComponent` instead?

**Review by styrelius — COMMENTED** [review]
Yay! ✨  I think it looks really good but not sure I should be the one to approve since I'm not that familiar with this repo? 🤔

File: src/Parsing/parseStoryData.ts [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 
-  return tooltips.map(parseTooltip)
+  return tooltips.map(parseTooltip(translate))
 }
 
-const parseTooltip = (element: Element) => {
-  const title = (
-    element.getElementsByTagName('Title')[0].textContent || ''
+const parseTooltip = (translate: Translator) => (element: Element) => {
+  const title = translate(
```
Comment (gustaveen) [inline src/Parsing/parseStoryData.ts:14]: This is one big file 😅  But as I understand you mostly added translate that resolves possible textKeys right 👌

**Review by gustaveen — DISMISSED** [review]
Great job with this!

File: src/Components/Common.tsx (hunk 1 above)
Comment (gustaveen) [reply src/Components/Common.tsx:20]: Good catch!

File: package.json [hunk 3]
```diff
@@ -21,11 +21,19 @@
     "@emotion/core": "^10.0.21",
     "@emotion/styled": "^10.0.17",
     "@hedviginsurance/brand": "^3.0.5",
+    "@hedviginsurance/textkeyfy": "^1.2.4",
```
Comment (styrelius) [inline package.json:24]: This one should be upgraded to `2.0.0` in order for this to work right?

**Review by gustaveen — APPROVED** [review]

**Comment by palmenhq** [comment]
I'm just gonna merge this despite travis being stuck

**Pull request merged**